doctrine/custom_dql_functions.rst

```diff
@@ -69,4 +69,23 @@ In Symfony, you can register your custom DQL functions as follows:
             ),
         ));
 
+.. note::
+
+    In case the `entity_managers` were named explicitly, configuring the functions with the 
+    orm directly will trigger the exception `Unrecognized option "dql" under "doctrine.orm"`.
+    The `dql` configuration block should be nested under the entity manager.
```

Comment:
`should be nested` -> `must be defined`

`under the entity manager.` -> `under the appropriate entity manager.` ? `under the specific entity manager.` ?  `under the named entity manager.` ?